src/DevChatter.Bot.Core/Automation/ChoseStartingBoxesAction.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using DevChatter.Bot.Core.Data.Model;
+using DevChatter.Bot.Core.Games.DealNoDeal;
+using DevChatter.Bot.Core.Systems.Chat;
+using DevChatter.Bot.Core.Util;
+
+
+namespace DevChatter.Bot.Core.Automation
+{
+    public class ChoseStartingBoxesAction : IIntervalAction
+    {
+        private readonly IClock _clock;
+        private DateTime _nextRunTime;
+        private IChatClient _chatClient;
+        private DealNoDealGame _dealNoDealGame;
+
+
+        public ChoseStartingBoxesAction(DealNoDealGame dealNoDealGame, IChatClient chatClient, IClock clock)
```

Comment:
You can `IClock clock = new SystemClock()` here giving it a default value, so you don't have to specify it everywhere.

Reply:
Fixed, that was sloppy from my side 👍